cmd/multiclusterapp.go

```diff
@@ -34,12 +36,21 @@ Example:
 `
 )
 
+var memberAccessTypes = []string{"owner", "member", "read-only"}
+
 type MultiClusterAppData struct {
+	ID      string
 	App     managementClient.MultiClusterApp
 	Version string
 	Targets string
 }
 
+type MemberData struct {
+	Name       string
```

Comment:
Member has different fields for groupPrincipalId and userPrincipalId https://github.com/rancher/types/blob/master/apis/management.cattle.io/v3/multi_cluster_app.go#L56
How do we differentiate between them here?

Reply:
It is shown in `memberType`, See L1002:
```
MemberType: strings.Title(fmt.Sprintf("%s %s", principal.Provider, principal.PrincipalType))
```